codeTranslations/ja.json

```diff
@@ -0,0 +1,8 @@
+{
+  "cmfcmf/d-s-l.searchBar.placeholderWithoutVersion": "検索...",
+  "cmfcmf/d-s-l.searchBar.placeholderWithVersion": "検索... [{version}]",
+  "cmfcmf/d-s-l.searchBar.noResults": "該当する結果はありません。",
+  "cmfcmf/d-s-l.searchBar.clearButtonTitle": "Clear",
+  "cmfcmf/d-s-l.searchBar.detachedCancelButtonText": "Cancel",
+  "cmfcmf/d-s-l.searchBar.submitButtonTitle": "Submit"
```

Comment:
It looks like three translations are missing here.

Reply:
our japanese translator was unsure if there would be enough room and proposed to keep EN words that are well understood

Reply:
I see, good to know 👍